src/main/java/com/rev_connect_api/controllers/HomeController.java

```diff
@@ -18,7 +18,7 @@ public class HomeController {
 
     @PostMapping
     public ResponseEntity<String> registerUser(@RequestBody User user) {
-        System.out.println("*************************Startig point*****************");
+        System.out.println("*************************Starting point*****************");
```

Comment:
You can remove this

Reply:
i just wanna keep this for testing, in my next commit i will do this